.gitignore

```diff
@@ -0,0 +1 @@
+coverage
```

Comment:
Note to review git ignore with broader team so we're all on same page as to what this functionality is doing.